packages/form/src/form.vue

```diff
@@ -50,7 +49,7 @@ function useFormLabelWidth() {
 
   function deregisterLabelWidth(val: number) {
     const index = getLabelWidthIndex(val)
-    potentialLabelWidthArr.value.splice(index, 1)
+    index > -1 && potentialLabelWidthArr.value.splice(index, 1)
```

Comment:
@kooriookami Will, with this line, the concern is certainly unnecessary, because if this value was not found, the function won't be invoked anyway.